test/extended/builds/forcepull.go

```diff
@@ -53,6 +59,11 @@ func doTest(bldPrefix, debugStr string, same bool, oc *exutil.CLI) {
 	// dump tags/hexids for debug
 	_, err = exutil.DumpAndReturnTagging(tags)
 	o.ExpectWithOffset(1, err).NotTo(o.HaveOccurred())
+
+	if bldPrefix == buildPrefixFC || bldPrefix == buildPrefixTC {
+		err = oc.AsAdmin().Run("adm").Args("policy", "remove-cluster-role-from-user", "system:build-strategy-custom", oc.Username()).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+	}
```

Comment:
I think this should be setup in a defer block so we run it even if the test fails.  same for the one below.